samples/drivers/spi_flash_at45/overlay-pm.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_DEVICE_POWER_MANAGEMENT=y
```

Comment:
Why not just move this line to `boards/nrf9160dk_nrf9160.conf` instead of introducing `overlay-pm.conf`?

Reply:
I think it will be nice show to user how select the LPM. It is easy to anyone understand how to select and test the feature. Besides, it is better for run tests. I took liberty to add another one for flash page layout.